build.gradle

```diff
@@ -110,7 +125,6 @@ subprojects { subproject ->
   }
 
   if (RELEASE_MODULES.contains(subproject.name)) {
-    subproject.apply from: "${rootDir}/gradle/gradle-bintray.gradle"
```

Comment:
should this be `subproject.apply from: "${rootDir}/gradle/publish.gradle"` ?

Reply:
Adding it throws compile time error. But, it kind os sounds right to have this line instead.

Reply:
Yes, `subproject.apply from: "${rootDir}/gradle/publish.gradle"` needs to be here.